### ISSUE TITLE
Ubuntu 16.04 - Failed to connect to bus: No such file or directory

### ISSUE THREAD

**Issue by igolman** [test-kitchen/dokken-images#101]
## :ghost: Brief Description

`systemd` is not available on `Ubuntu 16.04`

## Version

latest Docker Image from [DockerHub](https://hub.docker.com/r/dokken/ubuntu-16.04/tags)

## Environment

* Mac OSX Monterey `v12.6.8`
* Docker Desktop `v4.22.0`

## Scenario

Attempt to work with `systemd` on `Ubuntu 16.04`

## Steps to Reproduce

Consider simple `.kitchen.yml` config:
```yml
---
driver:
  name: dokken
  provision: true
  use_sudo: false
  privileged: true
  pull_chef_image: true
  pull_platform_image: true
  clean_dokken_sandbox: true
  use_cache: true
  pid_one_command: /bin/systemd

transport:
  name: dokken

provisioner:
  name: dokken

platforms:
  - name: ubuntu-16.04
    driver:
      image: dokken/ubuntu-16.04
  - name: ubuntu-18.04
    driver:
      image: dokken/ubuntu-18.04

suites:
  - name: default
```

## Expected Result

`systemctl status`  doesn't report `Failed to connect to bus: No such file or directory` error...

## Actual Result
```
$ kitchen list
Instance             Driver  Provisioner  Verifier  Transport  Last Action    Last Error
default-ubuntu-1604  Dokken  Dokken       Busser    Dokken     <Not Created>  <None>
default-ubuntu-1804  Dokken  Dokken       Busser    Dokken     <Not Created>  <None>
```

* Ubuntu 16.04 reports error:
```
$ kitchen create default-ubuntu-1604 && kitchen login default-ubuntu-1604
-----> Starting Test Kitchen (v3.5.0)
-----> Creating <default-ubuntu-1604>...
/opt/cinc-workstation/embedded/lib/ruby/gems/3.1.0/gems/lockfile-2.1.3/lib/lockfile.rb:308: warning: finalizer references object to be finalized
/opt/cinc-workstation/embedded/lib/ruby/gems/3.1.0/gems/lockfile-2.1.3/lib/lockfile.rb:308: warning: finalizer references object to be finalized
       Creating kitchen sandbox at /Users/dkd-golman/.dokken/kitchen_sandbox/ec676a802f-default-ubuntu-1604
       Creating verifier sandbox at /Users/dkd-golman/.dokken/verifier_sandbox/ec676a802f-default-ubuntu-1604
       Building work image..
       Creating container ec676a802f-default-ubuntu-1604
       Finished creating <default-ubuntu-1604> (0m36.79s).
-----> Test Kitchen is finished. (0m37.54s)
root@dokken:/# dpkg -l | grep systemd
ii  libsystemd0:amd64         229-4ubuntu21.31  amd64             systemd utility library
ii  systemd                   229-4ubuntu21.31  amd64             system and service manager
ii  systemd-sysv              229-4ubuntu21.31  amd64             system and service manager - SysV links
root@dokken:/# ps aux |grep systemd
root         1  0.2  0.0  36436  1876 ?        Ss   17:56   0:00 /bin/systemd
root        26  0.0  0.0  11288  1016 pts/0    S+   17:56   0:00 grep --color=auto systemd
root@dokken:/# systemctl status
Failed to connect to bus: No such file or directory
root@dokken:/# logout
```
* Ubuntu 18.04 just fine:
```
$ kitchen create default-ubuntu-1804 && kitchen login default-ubuntu-1804
-----> Starting Test Kitchen (v3.5.0)
-----> Creating <default-ubuntu-1804>...
/opt/cinc-workstation/embedded/lib/ruby/gems/3.1.0/gems/lockfile-2.1.3/lib/lockfile.rb:308: warning: finalizer references object to be finalized
/opt/cinc-workstation/embedded/lib/ruby/gems/3.1.0/gems/lockfile-2.1.3/lib/lockfile.rb:308: warning: finalizer references object to be finalized
       Creating kitchen sandbox at /Users/dkd-golman/.dokken/kitchen_sandbox/ec676a802f-default-ubuntu-1804
       Creating verifier sandbox at /Users/dkd-golman/.dokken/verifier_sandbox/ec676a802f-default-ubuntu-1804
       Building work image..
       Creating container ec676a802f-default-ubuntu-1804
       Finished creating <default-ubuntu-1804> (0m3.13s).
-----> Test Kitchen is finished. (0m3.86s)
root@dokken:/# dpkg -l | grep systemd
ii  dbus-user-session         1.12.2-1ubuntu1.4 amd64             simple interprocess messaging system (systemd --user int
ii  libnss-systemd:amd64      237-3ubuntu10.57  amd64             nss module providing dynamic user and group name resolut
ii  libpam-systemd:amd64      237-3ubuntu10.57  amd64             system and service manager - PAM module
ii  libsystemd0:amd64         237-3ubuntu10.57  amd64             systemd utility library
ii  networkd-dispatcher       1.7-0ubuntu3.5    all               Dispatcher service for systemd-networkd connection statu
ii  systemd                   237-3ubuntu10.57  amd64             system and service manager
ii  systemd-sysv              237-3ubuntu10.57  amd64             system and service manager - SysV links
root@dokken:/# ps aux |grep systemd
root         1  0.8  0.1  76972  8316 ?        Ss   17:57   0:00 /bin/systemd
root        25  0.4  0.1  78196  9632 ?        S<s  17:57   0:00 /lib/systemd/systemd-journald
systemd+    32  0.2  0.0  70500  4804 ?        Ss   17:57   0:00 /lib/systemd/systemd-resolved
message+    37  0.1  0.0  49936  4084 ?        Ss   17:57   0:00 /usr/bin/dbus-daemon --system --address=systemd: --nofork --no
root       106  0.0  0.0  61700  2824 ?        Ss   17:57   0:00 /lib/systemd/systemd-timedated
root       131  0.0  0.0  11472  1104 pts/0    S+   17:57   0:00 grep --color=auto systemd
root@dokken:/# systemctl status
● dokken
    State: running
     Jobs: 0 queued
   Failed: 0 units
    Since: Wed 2023-08-23 17:57:20 UTC; 21s ago
   CGroup: /
           ├─init.scope
           │ ├─  1 /bin/systemd
           │ ├─109 /bin/bash -login -i
           │ ├─132 systemctl status
           │ └─133 (pager)
           └─system.slice
             ├─cron.service
             │ └─35 /usr/sbin/cron -f
             ├─systemd-timedated.service
             │ └─106 /lib/systemd/systemd-timedated
             ├─networkd-dispatcher.service
             │ └─36 /usr/bin/python3 /usr/bin/networkd-dispatcher --run-startup-triggers
             ├─systemd-journald.service
             │ └─25 /lib/systemd/systemd-journald
             ├─systemd-resolved.service
             │ └─32 /lib/systemd/systemd-resolved
             └─dbus.service
               └─37 /usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
root@dokken:/# logout
```

**Comment by damacus** [comment]
Closing as we're not going to fix Ubuntu 16.04 issues. I highly suggest updating to a newer OS. 

If that fails we'll look into patches submitted.

**Comment by igolman** [comment]
Thanks for replying!

> I highly suggest updating to a newer OS.
Sure. However, I suppose I'm not the only one who due to cirumstances have now a few production servers running 16.04 with ESM subscription...

Thank you.
